docs/system-overview.rst

```diff
@@ -0,0 +1,84 @@
+Proctoring System Overview
+===========================
+This document outlines the components involved in the edX proctoring system. It should
+serve as a guide to orient developers on what components exist, how they interact, and 
+where to find the source code in our platform.
+
+.. contents::
+
+System Components
+------------------
+
+.. image:: images/components.png
+
+Proctored Exam Views
+^^^^^^^^^^^^^^^^^^^^
+
+Interstitial views within a section that are shown to in place of the actual
+exam content. They are used to walk a leaner through setup steps
+and display the state of the current attempt when the exam is not in progress.
```

Comment:
I think you're separating two things between this and the following paragraph: the goal of the code and its implementation. That's a good approach for documentation. I would expand on it, though, and be more explicit about the separation you're making, e.g. 
> The goal of these interstitials is to guide the learner through the many steps of attempting a proctored exam, where at many steps there are critical details about what the learner needs to do or understand. As such some of the copy on these interstitials is very important.